5.x/crud-how-to.md

```diff
@@ -586,6 +586,80 @@ This is in all aspects similar to [BelongsToMany](#belongstomany) relation, the
 
 Please read the relationship [BelongsToMany](#belongstomany) documentation, everything is the same in regards to fields definition and Backpack will take care of the morphable relation saving.
 
+<a name="morphto"></a>
+#### MorphTo (n-1 relationship)
+
+Using this relation type Backpack will automatically manage for you both `_type` and `_id` fields of this relation. 
+Lets take as example the `comments` that can be either form `videos` or `posts`. 
+- You should have in your `CommentModel` the `morphTo` relation setup.
+- You table should have the morph columns `commentable_type` and `commentable_id` properly setup.
+```php
+// in CommentCrudController you can add the morphTo fields by naming the field the morphTo relation name
+CRUD::field('commentable')
+    ->addMorphOption('App\Models\Video');
+    ->addMorphOption('App\Models\Post');
+```
+This will generate two inputs:
+1 - A select with two options `Video` and `Post` as the `morph type field`. 
+2 - A second select that will have the options for both `Video` and `Post` models.
+
+In a real world scenario you will have other needs like using `ajax` for the `models field select` when there are alot of entries, or to customize the inputs size etc.
+
+```php
+// ->addMorphOption(string $model/$morphMapName, string $labelInSelect, array $options)
+CRUD::field('commentable')
+    ->addMorphOption('App\Models\Video')
+    ->addMorphOption('App\Models\Post', 'Posts', [
+        [
+            'data_source'          => backpack_url('comment/fetch/post'),
+            'minimum_input_length' => 2,
+            'placeholder'          => 'select an amazing post',
+            'method'               => 'POST',
+            'attribute'            => 'title',
+        ]
+    ]);
+
+// by defining `data_source` you are telling Backpack that the `Posts` select should be an ajax select.
+```
+In this scenario the same two selects would be generated, but behind the scenes Backpack will switch between the `regular Video select` and the `ajax Post select`.
```

Comment:
```suggestion
In this scenario the same two selects would be generated, but for the Post, your admin see an AJAX field, instead of a static one, use POST instead of GET etc.
```